.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ on:
   pull_request:
 
 env:
-  DOTNET_VERSION: 5.0.100
+  DOTNET_VERSION: 6.0.x
```

Comment:
Deterministic version?

Reply:
This is copied from the official template, I think there is no problem.